app/src/main/java/com/andrew/tasky/agenda/data/util/ReminderTimeConversion.kt

```diff
@@ -39,21 +42,33 @@ object ReminderTimeConversion {
         }
     }
 
-    fun toEpochSecond(startTimeEpochSecond: Long, reminderTime: ReminderTime): Long {
-        val startTime = LocalDateTime.ofEpochSecond(
-            startTimeEpochSecond, 0, ZoneOffset.UTC
-        )
+    fun toEpochSecond(startLocalDateTime: LocalDateTime, reminderTime: ReminderTime): Long {
         return when (reminderTime) {
             ReminderTime.TEN_MINUTES_BEFORE ->
-                startTime.minusMinutes(10).toEpochSecond(ZoneOffset.UTC)
+                ZonedDateTime.of(
+                    startLocalDateTime.minusMinutes(10),
+                    TimeZone.getDefault().toZoneId()
+                ).toEpochSecond()
             ReminderTime.THIRTY_MINUTES_BEFORE ->
-                startTime.minusMinutes(30).toEpochSecond(ZoneOffset.UTC)
+                ZonedDateTime.of(
+                    startLocalDateTime.minusMinutes(30),
+                    TimeZone.getDefault().toZoneId()
+                ).toEpochSecond()
             ReminderTime.ONE_HOUR_BEFORE ->
-                startTime.minusHours(1).toEpochSecond(ZoneOffset.UTC)
+                ZonedDateTime.of(
+                    startLocalDateTime.minusHours(1),
+                    TimeZone.getDefault().toZoneId()
+                ).toEpochSecond()
             ReminderTime.SIX_HOURS_BEFORE ->
-                startTime.minusHours(6).toEpochSecond(ZoneOffset.UTC)
+                ZonedDateTime.of(
+                    startLocalDateTime.minusHours(6),
+                    TimeZone.getDefault().toZoneId()
+                ).toEpochSecond()
             ReminderTime.ONE_DAY_BEFORE ->
-                startTime.minusDays(1).toEpochSecond(ZoneOffset.UTC)
+                ZonedDateTime.of(
+                    startLocalDateTime.minusDays(1),
+                    TimeZone.getDefault().toZoneId()
+                ).toEpochSecond()
```

Comment:
All these calls are very similar, could it make sense to create a helper function that just takes in the amount of time you want to subtract?
Also, you're returning the timestamp in seconds again instead of milliseconds